lib/email_auth.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+#email_auth.py
+#
+#Combine standard login with option of OAuth2.0
+#
+from httplib2 import Http
+from oauth2client import file, client, tools
+from googleapiclient.discovery import build
+from lib.config_file import open_config
+import smtplib
+import base64
+from imaplib import IMAP4_SSL
+
+
+def email_authentication():
+   config = open_config()
+   email_config = config['email']
+   SCOPES = email_config["gmailUrl"]
```

Comment:
since this is a constant, just pull it out into a constant field outside of the methods and refer to it as such whenever you'd normally refer to SCOPES

Reply:
pulled out